validator/slashing-protection/local/standard-protection-format/import.go

```diff
@@ -0,0 +1,127 @@
+package interchangeformat
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/validator/db"
+	"github.com/prysmaticlabs/prysm/validator/db/kv"
+)
+
+// ImportStandardProtectionJSON takes in EIP-3076 compliant JSON file used for slashing protection
+// by eth2 validators and imports its data into Prysm's internal representation of slashing
+// protection in the validator client's database. For more information, see the EIP document here:
+// https://eips.ethereum.org/EIPS/eip-3076.
+func ImportStandardProtectionJSON(ctx context.Context, validatorDB db.Database, r io.Reader) error {
+	encodedJSON, err := ioutil.ReadAll(r)
+	if err != nil {
+		return errors.Wrap(err, "could not read slashing protection JSON file")
+	}
+	interchangeJSON := &EIPSlashingProtectionFormat{}
+	if err := json.Unmarshal(encodedJSON, interchangeJSON); err != nil {
+		return errors.Wrap(err, "could not unmarshal slashing protection JSON file")
+	}
+
+	attestingHistoryByPubKey := make(map[[48]byte]kv.EncHistoryData)
+	proposalHistoryByPubKey := make(map[[48]byte]kv.ProposalHistoryForPubkey)
+	for _, validatorData := range interchangeJSON.Data {
+		pubKey, err := pubKeyFromHex(validatorData.Pubkey)
+		if err != nil {
+			return fmt.Errorf("%s is not a valid public key: %v", validatorData.Pubkey, err)
+		}
+		// Parse and transform the signed attestation data from the JSON
+		// file into the internal Prysm representation of attesting history.
+		attestingHistory, err := parseSignedAttestations(ctx, validatorData.SignedAttestations)
+		if err != nil {
+			return errors.Wrapf(err, "could not parse signed attestations in JSON file for key %#x", pubKey)
+		}
+		attestingHistoryByPubKey[pubKey] = *attestingHistory
```

Comment:
We should handle multiple pubkeys in the imported json here

Reply:
+1 and for proposal